volume.go

```diff
@@ -359,3 +359,23 @@ func (v *Volume) SetVolumeAccessModeLimit(mode string) error {
 	err = v.client.getJSONWithRetry(http.MethodPost, path, payload, nil)
 	return err
 }
+
+// SetSnapshotSecurityParam defines type for snapshot retention period in min parameter for method SetSnapshotSecurity
+type SetSnapshotSecurityParam struct {
+	RetentionPeriodInMin string `json:"retentionPeriodInMin"`
+}
+
+// SetSnapshotSecurity set retention period in min on snapshot
+func (v *Volume) SetSnapshotSecurity(retentionPeriodInMin string) error {
```

Comment:
are the unit tests covered for this new function?

Reply:
added the unit test